res/themes/dark/css/_dark.scss

```diff
@@ -213,9 +213,10 @@ $breadcrumb-placeholder-bg-color: #272c35;
 
 $user-tile-hover-bg-color: $header-panel-bg-color;
 
-$message-body-panel-bg-color: #21262c82;
-$message-body-panel-icon-bg-color: #8e99a4;
-$message-body-panel-fg-color: $primary-fg-color;
+$message-body-panel-fg-color: $secondary-fg-color;
+$message-body-panel-bg-color: $voice-record-waveform-bg-color;
```

Comment:
Surely `$voice-record-waveform-bg-color` is "more specific", and so the actual colour value might go here, and then `$voice-record-waveform-bg-color` might be set to `$message-body-panel-bg-color` (inverting what you have here)?

Reply:
hmm, yea. I have somewhat been debating whether or not after https://github.com/matrix-org/matrix-react-sdk/pull/5970 lands if we should consolidate styles, but the components are just slightly too different to be the same...

Reply:
switching the variable order around is blocked on https://github.com/matrix-org/matrix-react-sdk/pull/5970